core/store/migrate/migrations/0194_evm_schema.sql

```diff
@@ -1,6 +1,6 @@
 -- +goose Up
 CREATE SCHEMA evm;
-SET search_path TO evm,public;
+SET search_path TO public,evm;
```

Comment:
i believe it's safe to edit this in place rather add an explicit migration. running make testdb works with the example provided by @jinhoonbang

Reply:
hmm but the change is on the up. If this has already been ran anywhere I don't believe it would receive the necessary update. If 194 was added not too long ago prior to any releases this may be okay. Or if this was part of the down. Otherwise not sure how it would affect existing data besides just the rollback for tests

Reply:
yeah, i think you are right. thank you. split into a new migration, while also fixing the down